primitives/api/test/Cargo.toml

```diff
@@ -21,7 +21,7 @@ sp-consensus = { version = "0.10.0-dev", path = "../../consensus/common" }
 sc-block-builder = { version = "0.10.0-dev", path = "../../../client/block-builder" }
 codec = { package = "parity-scale-codec", version = "3.0.0" }
 sp-state-machine = { version = "0.13.0", path = "../../state-machine" }
-trybuild = "1.0.60"
+trybuild = "1.0.74"
```

Comment:
https://github.com/paritytech/substrate/pull/13073 just merged and changed some of the pallet-ui files to `and $N others`.  
Please rebase to restart the CI and see if that works or needs reverting.